astronomer/providers/amazon/aws/example_dags/example_redshift_cluster_management.py

```diff
@@ -94,20 +98,20 @@ def create_redshift_cluster_snapshot() -> None:
         ],
     )
 
-    while True:
-        if get_snapshot_status() == "available":
-            break
+    while get_snapshot_status() != "available":
         time.sleep(30)
```

Comment:
probably worth adding a log line, to say we are sleeping, otherwise it will look like if the job is hanging

Reply:
yes, makes sense, added everywhere a log for the same.